docker/dataverse-k8s/payara/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Copyright 2019 Forschungszentrum Jülich GmbH
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+FROM payara/server-full:5.201
```

Comment:
I love this. Finally, we're starting from an app server image instead of downloading it and unzipping it!